tests/30rooms/30history-visibility.pl

```diff
@@ -96,8 +96,8 @@ sub test_history_visibility
 }
 
 foreach my $i (
-   [ "Guest", sub { guest_user_fixture( with_events => 1 ) } ],
-   [ "Real", sub { local_user_fixture( with_events => 1 ) } ]
+   [ "Guest", sub { guest_user_fixture() } ],
+   [ "Real", sub { local_user_fixture() } ]
```

Comment:
`local_user_fixture` *does* take a with-events param: it makes it call `/events` to initialise an `eventstream_token` field in the user... which we weren't using.